docs/apps/qiime.md

```diff
@@ -0,0 +1,96 @@
+# QIIME
+
+## Description
+
+QIIME (Quantitative Insights Into Microbial Ecology) is a package for comparison and analysis of microbial communities, 
+primarily based on high-throughput amplicon sequencing data (such as SSU rRNA) generated on a variety of platforms, 
+but also supporting analysis of other types of data (such as shotgun metagenomic data). QIIME takes users from their 
+raw sequencing output through initial analyses such as OTU picking, taxonomic assignment, and construction of 
+phylogenetic trees from representative sequences of OTUs, and through downstream statistical analysis, visualization, 
+and production of publication-quality graphics.
+
+On 2017 a totally rewritten version of Qiime: Qiime2 was released. The development of the original Qiime version has stopped. 
+At the moment only Qiime2 is available in Puhti.
+
+
+## Available
+
+-   Puhti: qiime2-2019.7  
+
+
+## Usage
+
+In Puhti, QIIME2 can be taken in use as a _bioconda_ environment:
+
+```text
+module load bioconda
+conda env list
+source activate qiime2-2019.7
+source tab-qiime 
+```
+
+After that you can start Qiime2 with command:
+```text
+qiime
+```
+
+Please check Qiime2 home page for more instructions.
+
+Note that many Qiime tasks involve heavy computing. Thus these tasks should be executed as
+batch jobs.
+
+For example the batch job script below runs the denoising step of the
+[QIIME moving pictures tutorial](https://docs.qiime2.org/2019.7/tutorials/moving-pictures/#option-1-dada2 )
+as a batch job using eight cores.
+
+```text
+#!/bin/bash
+#SBATCH --job-name=qiime_denoise
+#SBATCH --account=project_200XXXX 
```

Comment:
These should be --account=<project> like in, e.g., https://docs.csc.fi/#computing/running/creating-job-scripts/

People whose project do not follow this new style where confused when the documentation assumed the form is project_N

Reply:
Project definitions fixed